### PR TITLE
Allocations: New allocations panel

### PR DESCRIPTION
**Pull request by PeterMPhillips**
Fixes the form fields, the available funds message, and provide a help dialog. ~When the `budget-details` branch get merged I will redirect this to `dev`~

## Funds in vault greater than remaining in budget

Funds in vault greater than remaining in budget

## Funds in vault less than remaining in budget

Funds in vault less than remaining in budget

### CODE REVIEW

File: apps/allocations/app/components/Panel/NewAllocation.js [hunk 1]
```diff
@@ -171,86 +181,90 @@ class NewAllocation extends React.Component {
...
-                value={amountValue}
-                onChange={this.changeField}
-                wide={true}
-                css={{ borderRadius: '4px 0px 0px 4px' }}
-              />
-              <CurrencyBox>{tokenValue.symbol}</CurrencyBox>
-            </InputGroup>
+            <Help hint="Available funds">
+              Available funds displays the amount of funds remaining within this allocation’s budget.
+              If this amount exceeds the organization’s vault funds, an accordingly adjusted value will be shown here.
```
Comment (chadoh) [inline apps/allocations/app/components/Panel/NewAllocation.js]: @javieralaves sorry I didn't leave feedback on this sooner! Now that I see it here, I want to propose two possible simplifications to this help text.

1. Only show the help icon when the available funds are less than the remaining amount in the budget, as that's _probably_ when a user will be confused. Then, the help text can say: "There's {remainingBudget} left in this budget until {periodEndDate}, but only {inVault} left in the organization's vault."
2. Always show the help icon, and when the vaults funds are the limiting factor, show the same phrase suggested above. When the vault funds aren't the limiting factor, just say the first part of the phrase, "There's {remainingBudget} left in this budget until {periodEndDate}."

What do you think?

Comment (chadoh) [reply apps/allocations/app/components/Panel/NewAllocation.js]: I spoke with @javieralaves in Keybase, and he thought Option 2 above sounded good

Let's change this help text, @PeterMPhillips

**Comment by chadoh** [comment]
@stellarmagnet @javieralaves design review the GIFs if you can, please!

**Comment by chadoh** [comment]
Sorry, trying to settle the books for AutarkLabs/product#6; design review can be done in our continuous deploy env soon enough!

**Comment by javieralaves** [comment]
Looking good!

**Pull request merged**